src/language-js/print/ternary.js

```diff
@@ -174,14 +174,23 @@ function shouldExtraIndentForConditionalExpression(path) {
     return false;
   }
 
-  let ancestorCount = 1;
-  let ancestor = path.getParentNode(ancestorCount);
-  if (!ancestor) {
-    return false;
-  }
-  while (isChainElement(ancestor)) {
+  let ancestor;
+  let ancestorCount = 0;
+  do {
     ancestor = path.getParentNode(++ancestorCount);
-  }
+    const getAncestorName = () =>
+      path.callParent((path) => path.getName(), ancestorCount - 1);
+    if (
+      ancestor == null ||
+      (isCallExpression(ancestor) && getAncestorName() !== "callee") ||
+      (isMemberExpression(ancestor) &&
+        ancestor.computed &&
+        getAncestorName() !== "object")
```

Comment:
I think we can use the same condition on line 211

Reply:
I'll try

Reply:
Check https://github.com/prettier/prettier/pull/10266/commits/a32e498ecb4c0bdca381caa81296959c8a4a0da0, I think it's more clear

Reply:
Thank you!